api/chaos/experimentation/v1/experiment_run_config_pair_details.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package clutch.chaos.experimentation.v1;
+
+import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
+
+import "chaos/experimentation/v1/form.proto";
+
+option go_package = "experimentationv1";
+
+message ExperimentRunConfigPairDetails {
```

Comment:
I think it would be fine to call this 'ExperimentRunDetails'

Reply:
or just ExperimentDetails?

Reply:
I like ExperimentRunDetails (experiment itself is a an extremely overloaded word)